cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java

```diff
@@ -18,6 +18,7 @@
     String RUN_ENV = "runEnv";
     String HOOKS = "hooks";
     String NODE_NAME_KEY = "nodeName";
+    String EXECUTABLE_TYPE = "executableType";
```

Comment:
IMHO the correct place for `EXECUTABLE_TYPE` type would be in `LnaguageEventData`